pkg/client/generated/openebs.io/runtask/v1beta1/informer/externalversions/openebs.io/interface.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by informer-gen. DO NOT EDIT.
+
+package openebs
+
+import (
+	internalinterfaces "github.com/openebs/maya/pkg/client/generated/openebs.io/runtask/v1beta1/informer/externalversions/internalinterfaces"
+	v1alpha1 "github.com/openebs/maya/pkg/client/generated/openebs.io/runtask/v1beta1/informer/externalversions/openebs.io/v1alpha1"
+)
```

Comment:
Having complex auto-gen `informer and listers` path will be difficult to manage ( too long and redundant )?

Reply:
This is a good point. Not sure how to handle this.
I am not sure why it refers to externalversions?
Can we avoid externalversions?
@prateekpandey14